tests/modules/propagate.c

```diff
@@ -302,6 +302,21 @@ int propagateTestIncr(RedisModuleCtx *ctx, RedisModuleString **argv, int argc)
     return REDISMODULE_OK;
 }
 
+int propagateTestVerbatim(RedisModuleCtx *ctx, RedisModuleString **argv, int argc) {
+    if (argc < 2){
+        RedisModule_WrongArity(ctx);
+        return REDISMODULE_OK;
+    }
+
+    long long replicate_num;
+    RedisModule_StringToLongLong(argv[1], &replicate_num);
+    /* Replicate the command verbatim for the specified number of times. */
+    for (long long i = 0; i < replicate_num; i++)
+        RedisModule_ReplicateVerbatim(ctx);
```

Comment:
is it valid use of RedisModule_ReplicateVerbatim to call it more than once per command?
feels like maybe we'll want it to detect this case and return an error some day, and maybe here we should use the other API.

Reply:
I was using RM_Replicate() to reproduce it, but it will allocate huge memory due to use `moduleCreateArgvFromUserFormat()` to create replicate argv, but RM_ReplicateVerbatim() doesn't, it just replicate ctx->client->argv.

Reply:
not sure i fully understand (too busy), maybe there's another way. but anyway, this is just odd, but for a test it's ok.

Reply:
yes, I do agree it's odd, but in order to validate memory spikes we need to avoid allocating too much memory before replicating to the replicas, which may take up huge memory as the global replication buffer without trimed.
RM_Replicate() -> moduleCreateArgvFromUserFormat() (here it will make the copy of the argv that I will pass in, just like we passed it to the global replication buffer).